packages/forms-web-app/src/routes/index.js

```diff
@@ -6,8 +6,10 @@ const appellantSubmissionRouter = require('./appellant-submission');
 const eligibilityRouter = require('./eligibility');
 const homeRouter = require('./home');
 const cookieRouter = require('./cookies');
+const guidancePagesRouter = require('./guidance-pages');
 
 router.use('/', homeRouter);
+router.use('/', guidancePagesRouter);
```

Comment:
Personal preference - could this be changed to:

`router.use(guidancePagesRouter);`

To me (and again, appreciate personal opinion), but this reads like there's two routers handling the same path, which feels like the second router shouldn't work. Appreciate it does in express, so feel free to ignore this. Some frameworks wouldn't like this ... and those are the ones I come from :D 

Again, feel free to ignore.

Reply:
I mainly went for consistency but I can see the benefit of it. Will update.